src/server/src/main/java/com/spotify/reaper/resources/view/NodesStatus.java

```diff
@@ -20,6 +20,7 @@
 import jersey.repackaged.com.google.common.collect.Maps;
 
 import java.math.BigDecimal;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.math.BigDecimal'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)